index.html

```diff
@@ -0,0 +1,132 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+    <link
+  rel="stylesheet"
+  href="https://use.fontawesome.com/releases/v5.8.1/css/all.css"
+  integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf"
+  crossorigin="anonymous"
+/>
+</head>
+<body>
+
+  
+
+    <!--header section-->
+        <header id="header">
+            <div class="container">
+                <nav id="nav">
+                    <h1 class="logo">Original Trombones</h1>
+                    <ul class="nav-items">
+                        <li class="item"><a href="#" class="link">Feature</a></li>
+                        <li class="item"><a href="#" class="link">How it Works</a></li>
+                        <li class="item"><a href="#" class="link">Pricing</a></li>
+                    </ul>
+                </nav>
+            </div>
+        </header>
+    <!--end header section-->
+
+    <!--main section-->
+        <main id="main" role="main">
+            <div class="container">
+                <div class="content-main">
+                        <h1 class="title">Handcrafted,home-made masterpieces</h1>
+                        <form class="form" action="">
+                           <input class="text" type="email" placeholder="Enter your email address" required>
+                           <input class="btn" type="submit" value="GET STARTED"> 
+                        </form>
+                        
+                </div>
+            </div>
+        </main>
+    <!--end of main section-->
+    <!--our services -->
+    <div class="container">
+     <article id="services">
+        <div class="icon"><i class="fa fa-3x fa-fire"></i></div>
+        <div class="content">
+            <h1 class="title">Premium Materials</h1>
+            <P class="text">Our trombones use the shiniest brass which is sourced locally. This will increase the longevity of your purchase.
+            </P>
+        </div>
+    </article>  
+    <article id="services">
+        <div class="icon"><i class="fa fa-3x fa-truck"></i></div>
+        <div class="content">
+            <h1 class="title">Fast Shipping</h1>
+            <P class="text">We make sure you recieve your trombone as soon as we have finished
+                making it. We also provide free returns if you are not satisfied.
+            </P>
+        </div>
+    </article> 
+    <article id="services">
+        <div class="icon">
+            <i class="fa fa-3x fa-battery-full" aria-hidden="true"></i>
+          </div>
+        <div class="content">
+            <h1 class="title">Premium Materials</h1>
+            <P class="text">For every purchase you make, we will ensure there are no damages or
+                faults and we will check and test the pitch of your instrument.
+            </P>
+        </div>
+    </article>  
+    </div>
+    
+
+<!--the end of our services-->
+
+
```

Comment:
- [OPTIOANAL ] You should review the spaces through your codes. Kindly review these space, and explain to students why its important to not add spaces in codes